### PR TITLE
fixed styling that broke pages

### PR DESCRIPTION
**Pull request by ghost**
I moved a p selector in WinnerPage.less so that it stopped impacting the p tags on other pages, causing the pages to not function properly.

### CODE REVIEW

**Review by rhiannonstanford — APPROVED** [review]
Nice clean up and good job finding that bug related to the p tag selector!

**Review by ashtilawat — COMMENTED** [review]
Qualified

**Review by ashtilawat — APPROVED** [review]

File: src/styles/less/WinnerPage.less [hunk 1]
```diff
@@ -3,106 +3,96 @@
...
-.winner-container{
-display: flex;
-justify-content: center;
-background: #292929;
-padding: 2%;
+.winner-container {
+  display: flex;
+  justify-content: center;
+  background: #292929;
+  padding: 2%;
```
Comment (BryanPAnderson) [inline src/styles/less/WinnerPage.less:13]: good bug fixes!

File: src/styles/less/WinnerPage.less [hunk 2]
```diff
@@ -3,106 +3,96 @@
...
+  left: 15%;
 }
-.winner-box .winner-boy-img{
+.winner-box .winner-boy-img {
   position: absolute;
   top: 45%;
-    display: flex;
-    justify-content: center;
+  display: flex;
+  justify-content: center;
```
Comment (BryanPAnderson) [inline src/styles/less/WinnerPage.less:29]: good catch on bug fix

File: src/styles/less/WinnerPage.less [hunk 3]
```diff
@@ -118,7 +108,13 @@ align-items: flex-end;
...
+  0% {
+    transform: scale(1);
+  }
+  50% {
+    transform: scale(1.1);
+  }
+  100% {
+    transform: scale(1);
+  }
+}
```
Comment (BryanPAnderson) [inline src/styles/less/WinnerPage.less:120]: good job working together to fix this

**Review by Yonathan-Admasu728 — COMMENTED** [review]
crucial fix. Thank you

**Pull request merged**